VirtoCommerce.ContentModule.Web/Scripts/blades/pages/edit-blog.js

```diff
@@ -45,26 +45,31 @@
     $scope.saveChanges = function () {
         blade.isLoading = true;
 
-        contentApi.saveWithMetadata({
-            contentType: blade.contentType,
-            storeId: blade.storeId,
-            folderUrl: ''
-        }, {
-            dynamicProperties: blade.currentEntity.dynamicProperties,
-            content: '',
-            name: getBlogBlobName()
-        },
-        function (data) {
-            blade.isLoading = false;
-            blade.origEntity = angular.copy(blade.currentEntity);
-            if (blade.isNew) {
-                $scope.bladeClose();
-                $rootScope.$broadcast("cms-statistics-changed", blade.storeId);
-            }
+        contentApi.createFolder(
```

Comment:
I think we need to check name uniqueness or something and handle it - now creating a blog with the same name as existing one results in console error, and maybe in blog rewriting.